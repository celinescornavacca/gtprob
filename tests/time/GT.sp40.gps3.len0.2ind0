((((((EB0,(EB1,EB2)),(FB0,(FB1,FB2))),(((AB0,(AB1,AB2)),((BB1,BB2),BB0)),(((HB1,HB2),HB0),((MB0,(MB1,MB2)),((NB1,NB2),NB0))))),(((IB1,IB2),IB0),((JB1,JB2),JB0))),(((CB0,(CB1,CB2)),((DB1,DB2),DB0)),((EA0,(EA1,EA2)),(FA0,(FA1,FA2))))),((((BA1,BA2),BA0),(((KB0,(KB1,KB2)),((LB1,LB2),LB0)),(GB0,(GB1,GB2)))),((((HA0,(HA1,HA2)),(((RA1,RA2),RA0),(((UA1,UA2),UA0),(VA0,(VA1,VA2))))),(((KA0,(KA1,KA2)),(((PA1,PA2),PA0),(QA0,(QA1,QA2)))),(GA0,(GA1,GA2)))),(((AA0,(AA1,AA2)),(((DA1,DA2),DA0),(((NA0,(NA1,NA2)),((OA1,OA2),OA0)),(((LA0,(LA1,LA2)),((MA1,MA2),MA0)),(((((WA1,WA2),WA0),(XA0,(XA1,XA2))),((SA0,(SA1,SA2)),((TA1,TA2),TA0))),(JA0,(JA1,JA2))))))),(((CA1,CA2),CA0),(((YA0,(YA1,YA2)),((ZA1,ZA2),ZA0)),((IA1,IA2),IA0)))))));
