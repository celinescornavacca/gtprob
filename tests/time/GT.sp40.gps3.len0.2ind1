(((((((IA1,IA2),IA0),((((XA1,XA2),XA0),((YA1,YA2),YA0)),((RA1,RA2),RA0))),((FB0,(FB1,FB2)),((GB1,GB2),GB0))),((FA0,(FA1,FA2)),(((MA1,MA2),MA0),((NA1,NA2),NA0)))),((((((BA1,BA2),BA0),(((ZA1,ZA2),ZA0),((AB1,AB2),AB0))),(AA0,(AA1,AA2))),(((VA0,(VA1,VA2)),(WA0,(WA1,WA2))),(((((DB1,DB2),DB0),((EB1,EB2),EB0)),(JA0,(JA1,JA2))),((JB0,(JB1,JB2)),((MB0,(MB1,MB2)),(NB0,(NB1,NB2))))))),((((HB0,(HB1,HB2)),(IB0,(IB1,IB2))),(UA0,(UA1,UA2))),(((PA0,(PA1,PA2)),((QA1,QA2),QA0)),(((SA0,(SA1,SA2)),(TA0,(TA1,TA2))),((((KB0,(KB1,KB2)),((LB1,LB2),LB0)),((HA1,HA2),HA0)),((GA1,GA2),GA0))))))),(((CA0,(CA1,CA2)),((DA1,DA2),DA0)),((((((BB1,BB2),BB0),((CB1,CB2),CB0)),((OA1,OA2),OA0)),(EA0,(EA1,EA2))),(((KA1,KA2),KA0),(LA0,(LA1,LA2))))));
