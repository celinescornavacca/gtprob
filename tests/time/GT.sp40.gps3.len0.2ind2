((((((XA1,XA2),XA0),(((IB1,IB2),IB0),(JB0,(JB1,JB2)))),(((LA1,LA2),LA0),((MA1,MA2),MA0))),(((AB0,(AB1,AB2)),((BB1,BB2),BB0)),(((((VA1,VA2),VA0),(WA0,(WA1,WA2))),(FA0,(FA1,FA2))),(((IA0,(IA1,IA2)),(((NA1,NA2),NA0),(((TA1,TA2),TA0),((UA1,UA2),UA0)))),(((((DA1,DA2),DA0),(((OA1,OA2),OA0),((PA1,PA2),PA0))),((CA1,CA2),CA0)),(((CB0,(CB1,CB2)),(DB0,(DB1,DB2))),(((GB1,GB2),GB0),((HB1,HB2),HB0)))))))),((((((((YA1,YA2),YA0),((ZA1,ZA2),ZA0)),(JA0,(JA1,JA2))),((((EB1,EB2),EB0),((FB1,FB2),FB0)),(((QA1,QA2),QA0),((RA0,(RA1,RA2)),((SA1,SA2),SA0))))),((AA1,AA2),AA0)),((((GA1,GA2),GA0),(HA0,(HA1,HA2))),((EA1,EA2),EA0))),((BA0,(BA1,BA2)),((KA0,(KA1,KA2)),((((KB1,KB2),KB0),((LB1,LB2),LB0)),(((MB1,MB2),MB0),((NB1,NB2),NB0)))))));
