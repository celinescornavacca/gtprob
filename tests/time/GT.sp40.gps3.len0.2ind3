(((((DA0,(DA1,DA2)),((RA0,(RA1,RA2)),(SA0,(SA1,SA2)))),((CB0,(CB1,CB2)),(DB0,(DB1,DB2)))),(((((BA0,(BA1,BA2)),(CA0,(CA1,CA2))),(AA0,(AA1,AA2))),(((HA0,(HA1,HA2)),(((AB1,AB2),AB0),(BB0,(BB1,BB2)))),(((IA1,IA2),IA0),(((YA1,YA2),YA0),((ZA1,ZA2),ZA0))))),((((((QA1,QA2),QA0),((GB0,(GB1,GB2)),((HB1,HB2),HB0))),((MA1,MA2),MA0)),((WA0,(WA1,WA2)),(((MB1,MB2),MB0),((NB1,NB2),NB0)))),(((((IB1,IB2),IB0),((JB1,JB2),JB0)),((OA1,OA2),OA0)),((((TA0,(TA1,TA2)),((UA0,(UA1,UA2)),(((KB1,KB2),KB0),((LB1,LB2),LB0)))),(LA0,(LA1,LA2))),(((JA1,JA2),JA0),((KA1,KA2),KA0))))))),(((((NA1,NA2),NA0),((((VA1,VA2),VA0),(((EB0,(EB1,EB2)),(FB0,(FB1,FB2))),((XA1,XA2),XA0))),((PA1,PA2),PA0))),(GA0,(GA1,GA2))),((EA0,(EA1,EA2)),(FA0,(FA1,FA2)))));
