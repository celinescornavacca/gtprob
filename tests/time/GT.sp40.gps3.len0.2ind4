((((((ZA1,ZA2),ZA0),(((MB1,MB2),MB0),((NB1,NB2),NB0))),((AB0,(AB1,AB2)),((BB1,BB2),BB0))),((((((MA0,(MA1,MA2)),(NA0,(NA1,NA2))),((XA0,(XA1,XA2)),((EB0,(EB1,EB2)),((FB0,(FB1,FB2)),((HB0,(HB1,HB2)),(((IB1,IB2),IB0),((JB1,JB2),JB0))))))),((AA1,AA2),AA0)),(((LA0,(LA1,LA2)),(((TA1,TA2),TA0),(UA0,(UA1,UA2)))),(((GB0,(GB1,GB2)),((KB0,(KB1,KB2)),(LB0,(LB1,LB2)))),(YA0,(YA1,YA2))))),((((GA1,GA2),GA0),((HA1,HA2),HA0)),((BA0,(BA1,BA2)),(((((OA1,OA2),OA0),((PA1,PA2),PA0)),(KA0,(KA1,KA2))),((CA0,(CA1,CA2)),(((IA1,IA2),IA0),((JA1,JA2),JA0)))))))),((((DA1,DA2),DA0),(EA0,(EA1,EA2))),((((QA0,(QA1,QA2)),(RA0,(RA1,RA2))),((SA0,(SA1,SA2)),((CB0,(CB1,CB2)),(DB0,(DB1,DB2))))),(((VA0,(VA1,VA2)),((WA1,WA2),WA0)),(FA0,(FA1,FA2))))));
