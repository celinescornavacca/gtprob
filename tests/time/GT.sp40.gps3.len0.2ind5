(((((((((PA1,PA2),PA0),((CB0,(CB1,CB2)),(DB0,(DB1,DB2)))),(((FA1,FA2),FA0),(((JB1,JB2),JB0),((KB1,KB2),KB0)))),(((DA0,(DA1,DA2)),((LA0,(LA1,LA2)),((MA1,MA2),MA0))),((YA0,(YA1,YA2)),((ZA1,ZA2),ZA0)))),((CA1,CA2),CA0)),(((((XA1,XA2),XA0),(((EB1,EB2),EB0),((FB1,FB2),FB0))),((QA1,QA2),QA0)),(BA0,(BA1,BA2)))),((((((NA1,NA2),NA0),(OA0,(OA1,OA2))),(((UA1,UA2),UA0),((AB0,(AB1,AB2)),((BB1,BB2),BB0)))),((VA0,(VA1,VA2)),((WA1,WA2),WA0))),((AA1,AA2),AA0))),(((((JA0,(JA1,JA2)),((KA1,KA2),KA0)),((EA1,EA2),EA0)),(((IA1,IA2),IA0),(((RA1,RA2),RA0),(SA0,(SA1,SA2))))),(((((((MB0,(MB1,MB2)),((NB1,NB2),NB0)),((LB1,LB2),LB0)),(GB0,(GB1,GB2))),(((HB1,HB2),HB0),(IB0,(IB1,IB2)))),((TA1,TA2),TA0)),((GA0,(GA1,GA2)),((HA1,HA2),HA0)))));
