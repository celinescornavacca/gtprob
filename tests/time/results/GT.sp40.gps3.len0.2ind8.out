-177.205677036
